packages/codegen-ui-react/lib/react-component-render-helper.ts

```diff
@@ -193,12 +193,14 @@ export function buildFixedLiteralExpression(
   const { value, type } = prop;
   switch (typeof value) {
     case 'number':
-      return factory.createNumericLiteral(value, undefined);
+      return factory.createNumericLiteral(value as number, undefined);
     case 'boolean':
       return value ? factory.createTrue() : factory.createFalse();
     case 'string':
-      return fixedPropertyWithTypeToLiteral(value, type);
     case 'object':
+      if (type !== 'object') {
```

Comment:
related to the stringified array issue on comment below. Workaround to reduce blast radius.